plugins/keepkey/keepkey.py

```diff
@@ -91,10 +92,24 @@ def __init__(self, parent, config, name):
             self.DEVICE_IDS = (keepkeylib.transport_hid.DEVICE_IDS +
                                keepkeylib.transport_webusb.DEVICE_IDS)
             self.device_manager().register_devices(self.DEVICE_IDS)
+            self.device_manager().register_enumerate_func(self.enumerate)
```

Comment:
Note that now in some cases the keepkey will be found twice during scans.
If the keepkey uses webusb as transport, then it will be found in both cases below:
https://github.com/Electron-Cash/Electron-Cash/blob/b7332d3348fd5963f0843c549e247c84e26f9e01/lib/plugins.py#L844
https://github.com/Electron-Cash/Electron-Cash/blob/b7332d3348fd5963f0843c549e247c84e26f9e01/lib/plugins.py#L847-L849
It seems that there is some issue with either python-keepkey or the keepkey firmware however, and one of these (the one found by `self.enumerate` is not always usable. (I am constantly getting `usb1.USBErrorAccess: LIBUSB_ERROR_ACCESS [-3]` but the errors are hidden from the user, only visible in the log)

I would be inclined to either use the custom enumerate functions for both the hid and webusb transports (which is what the trezor plugin does) from python-keepkey; or to only register the hid device id with the device_manager, i.e.:
```
self.device_manager().register_devices(keepkeylib.transport_hid.DEVICE_IDS)
```
However this actually does not work for me in practice due to above ("one of these ... is not always usable").